layouts/_default/baseof.html

```diff
@@ -54,14 +54,7 @@
   <meta name="google-site-verification" content="sBa9o-9RVlIHHlyeaHgIir20ZKcSuzLVgCCeXIMgKZg" />
 </head>
 
-{{ with $pageTitle }}
-
-<body vocab="http://schema.org/" typeof="WebPage" class="{{ $pageTitle }} {{ (string ( $.Site.Language) ) }}">
```

Comment:
Why put the page title into a `class` attribute? 🤔 💭

Reply:
I have no clue unfortunately. But it feels wrong.

Reply:
One option would just be to slugify it, along the lines of,
```
class="page-{{ $pageTitle  | urlize }}"
```

<https://gohugo.io/functions/urlize/>

That would get rid of the spaces between each word, and - by sticking a common word like "page" in front - it would avoid [CSS class issues if we had a page title that started with a number](https://stackoverflow.com/a/59058206).

I'm not sure why this was added but on the very rare chance that someone wanted to make blog-post specific CSS down the road, they could. 😜

Reply:
I like that idea, do you feel we should still concatenate the language ($.Site.Language) ? Title are usually localized, and could be use by itself to customize the css.

Reply:
I have removed the site.lang as the extra css class. I haven't found any usage.